pkg/skaffold/build/bazel/dependencies.go

```diff
@@ -103,20 +103,24 @@ func depToPath(dep string) string {
 	return strings.TrimPrefix(strings.Replace(strings.TrimPrefix(dep, "//"), ":", "/", 1), "/")
 }
 
-func findWorkspace(workingDir string) (string, error) {
+func findWorkspace(workingDir string) (string, string, error) {
 	dir, err := filepath.Abs(workingDir)
 	if err != nil {
-		return "", fmt.Errorf("invalid working dir: %w", err)
+		return "", "", fmt.Errorf("invalid working dir: %w", err)
 	}
 
 	for {
+		if _, err := os.Stat(filepath.Join(dir, "WORKSPACE.bazel")); err == nil {
+			return dir, "WORKSPACE.bazel", nil
+		}
+
```

Comment:
nit: is there reasoning for this taking precedence over `WORKSPACE`? Bazel docs seem to primarily mention `WORKSPACE`, rather than `WORKSPACE.bazel`

https://docs.bazel.build/versions/main/be/workspace.html

Reply:
according to this `WORKSPACE.bazel` takes precedence:
https://bazel.build/concepts/build-ref

quote: `Bazel also supports WORKSPACE.bazel file as an alias of WORKSPACE file. If both files exist, WORKSPACE.bazel is used.`

Reply:
awesome, thanks for pointing to this :)